content/exchange/artifacts/KnockKnock.yaml

```diff
@@ -34,7 +34,7 @@ sources:
         FROM Artifact.Generic.Utils.FetchBinary(ToolName="KnockKnock", IsExecutable='N')
       LET tempfolder <= tempdir()
         
-      LET bin <= SELECT * FROM unzip(filename=tool.OSPath[0],output_directory=tempfolder)
+      LET bin <= SELECT * FROM unzip(filename=tool.FullPath,output_directory=tempfolder)
```

Comment:
we are actually deprecating the use of FullPath - it should be ok to use OSPath everywhere now

Reply:
I was unable to use the artifact out of the box with `tool.OSPath[0]` - I also tried `tool[0].OSPath` which I saw in other artifacts. What would be the best way to have it configured?

Reply:
What version server are you using? The FullPath has been deprecated for a while

Reply:
I upgraded the server and it works out of the box. Thanks for the heads up!